src/app/payment-hub/transactions/incoming/incoming-transactions.component.html

```diff
@@ -16,8 +16,12 @@
   </mat-form-field> -->
 
   <mat-form-field fxFlex="20%">
-    <mat-label>Payer DFSP Id</mat-label>
-    <input matInput [formControl]="payerDfspId">
+    <mat-label>AMS Business Short Code</mat-label>
+    <mat-select [formControl]="payerDfspId">
+      <mat-option *ngFor="let code of amsCodes" [value]="code.value">
+        {{ code.option }}
```

Comment:
Similar to the previous comment, only Paybill shortcodes are applicable for `incoming transactions`